data_structures/hash_table/hash_table.py

```diff
@@ -0,0 +1,81 @@
+from linked_list import LinkedList
+
+
+class HashTable:
+    def __init__(self, max_size=1024):
+        """
+        This initiatates the hash table with empty linked lists in each bucket.
+        """
+
+        if type(max_size) is not int:
+            raise TypeError
+
+        linked_list = LinkedList()
+        self.max_size = max_size
+        self.buckets = [linked_list] * self.max_size
```

Comment:
This puts the same linked list in all the buckets, thus adding an item to one bucket will effectively add it to all of them.